docker-compose.yml

```diff
@@ -55,6 +55,7 @@ services:
     container_name: ops-backend
     ports:
       - "8080:8080"
+    command: ["python", "-m", "flask", "run", "--host=0.0.0.0", "--port=8080"]
```

Comment:
I know this is only used in dev. Do we prefer to simply use the flask engine in dev? We should keep some parity with live though, during testing, etc

Reply:
yes the flask dev web server provides better error message and context logging than the production web server - so this helps for development work